client/a8c-for-agencies/sections/marketplace/pressable-overview/lib/get-pressable-plan.ts

```diff
@@ -110,12 +111,65 @@ const PLAN_DATA: Record< string, PressablePlan > = {
 		visits: 3000000,
 		storage: 450,
 	},
+
+	// Pressable Enterprise plans
+	'pressable-enterprise-4': {
+		slug: 'pressable-enterprise-4',
+		install: 200,
+		visits: 4000000,
+		storage: 500,
+	},
+	'pressable-enterprise-5': {
+		slug: 'pressable-enterprise-5',
+		install: 250,
+		visits: 5000000,
+		storage: 550,
+	},
+	'pressable-enterprise-6': {
+		slug: 'pressable-enterprise-6',
+		install: 300,
+		visits: 6000000,
+		storage: 600,
+	},
+	'pressable-enterprise-7': {
+		slug: 'pressable-enterprise-7',
+		install: 350,
+		visits: 7000000,
+		storage: 700,
+	},
+	'pressable-enterprise-8': {
+		slug: 'pressable-enterprise-8',
+		install: 400,
+		visits: 8000000,
+		storage: 800,
+	},
+	'pressable-enterprise-9': {
+		slug: 'pressable-enterprise-9',
+		install: 450,
+		visits: 9000000,
+		storage: 900,
+	},
+	'pressable-enterprise-10': {
+		slug: 'pressable-enterprise-10',
+		install: 500,
+		visits: 10000000,
+		storage: 1000,
+	},
 };
 
 export default function getPressablePlan( slug: string ) {
-	return PLAN_DATA[ slug ];
+	return addPlanCategory( PLAN_DATA[ slug ] );
```

Comment:
Not a strong opinion, but maybe we could just add this to the object itself, to keep things more straight forward.

Reply:
I like that idea, keeps it simpler I suppose